web/index.js

```diff
@@ -1,3 +1,7 @@
+var Habitat = require('habitat');
+Habitat.load();
+var habitat = new Habitat();
```

Comment:
if we're still relying on `process.env` then you don't need this line; `Habitat.load()` does the "load-file-into-process.env" bit